tests/integration/test_charm.py

```diff
@@ -42,6 +50,7 @@ async def test_build_and_deploy(ops_test: OpsTest, request: pytest.FixtureReques
 async def test_given_gocert_when_tls_requirer_related_then_csr_uploaded_to_gocert(
```

Comment:
Can we either change the name of the test or have the part that checks for the cert itself in a different test case?

Reply:
Added to the name. I would prefer to keep the tests for a relation in one test if possible.